ethcore/src/engines/tendermint/mod.rs

```diff
@@ -260,8 +241,17 @@ impl Tendermint {
 		self.validators.contains(&*self.proposal_parent.read(), address)
 	}
 
-	fn is_above_threshold(&self, n: usize) -> bool {
-		n > self.validators.count(&*self.proposal_parent.read()) * 2/3
+	fn is_above_threshold(&self, n: usize) -> Result<(), EngineError> {
```

Comment:
Functions with `is_` prefix usually return `bool`. Maybe `check_` or `validate_if` would be a better name?